### ISSUE TITLE
Errors from @kubernetes/client-node returned as "{}"

### ISSUE THREAD

**Issue by koshkarov** [dot-i/k8s-operator-node#7]
**What is wrong:** Errors from @kubernetes/client-node returned as "{}"
**Why:** Properties of the Error object are not enumerable. 
From the documentation of JSON.stringify(),
`For all the other Object instances (including Map, Set, WeakMap and WeakSet), only their enumerable properties will be serialized.`

**Possible solutions:**
https://stackoverflow.com/questions/18391212/is-it-not-possible-to-stringify-an-error-using-json-stringify

**Comment by dot-i** [comment]
Thanks for reporting! Fixed in `1.0.15`.

**Comment by koshkarov** [comment]
@dot-i Thanks Nico for a quick fix. Your library is very helpful and I appreciate the work you've done!

**Comment by koshkarov** [comment]
@dot-i I've tested the fix and it doesn't work as expected. 

here is what I've got when got an authentication error from @kubernetes/client-node:
```
watching resource services.v1
watch on resource services.v1 failed: [object Object]
```

So I would say that this issue is not fixed. 

How to reproduce: 
1. Run operator without k8s authentication (kubeconfig, k8s token or anything else that is considered as authentication data for @kubernetes/client-node). 

ACTUAL RESULT: 
The error message is logged "watch on resource services.v1 failed: [object Object]" without error object itself. 

EXPECTED RESULT:
The error message is logged with the error object.
